examples/connectivity/plot_extract_regions_statistical_maps.py

```diff
@@ -0,0 +1,84 @@
+"""
+Region Extraction using a t-statistical maps (3D)
+=================================================
+
+This example shows how to extract regions or seperate the regions
+from the statistical maps.
+
+We use localizer t-statistic maps as an input image.
+
+The idea is to threshold an input for foreground objects using a
+function :func:`nilearn.regions.region_extractor.foreground_extraction`
+and extract objects using a function :func:`nilearn.regions.region_extractor.connected_component_extraction`.
+
+In this example, we also show how to use raw t-statistical value as a
+threshold and extract regions based on survived foreground objects.
+"""
+
+# Load localizer datasets - contrast/t maps
+from nilearn import datasets
+print(" -- Fetching t-statistic image from localizer datasets -- ")
+n_subjects = 3
+localizer_path = datasets.fetch_localizer_contrasts(
+    ['calculation (auditory cue)'], n_subjects=n_subjects, get_tmaps=True)
+
+# Foreground Extraction
+from nilearn.regions.region_extractor import foreground_extraction
+print(" -- Foreground Extraction -- ")
+# Foreground Extraction showing for two different types of strategies.
+# Here, we used thresholding_strategy='percentile' and None.
```

Comment:
This design is such that if user wants foreground objects based on t-value then there is no need of any strategy. It just thresholds with the input value. I agree it is not more explicit. May be I would remove the line ? or give more explanation ?